projects/packages/forms/src/blocks/contact-form/components/jetpack-field-controls.js

```diff
@@ -163,105 +157,260 @@ const JetpackFieldControls = ( {
 					onClick={ () => setAttributes( { required: ! required } ) }
 				/>
 			</BlockControls>
-
 			<InspectorControls>
 				<PanelBody title={ __( 'Manage Responses', 'jetpack-forms' ) }>
 					<JetpackManageResponsesSettings isChildBlock />
 				</PanelBody>
 				<PanelBody title={ __( 'Field Settings', 'jetpack-forms' ) }>
 					<>{ fieldSettings }</>
 				</PanelBody>
-				<PanelColorSettings
-					title={ __( 'Color', 'jetpack-forms' ) }
-					initialOpen={ false }
-					colorSettings={ colorSettings }
-				/>
-				<PanelBody title={ stylesPanelTitle } initialOpen={ false }>
-					<BaseControl>
+			</InspectorControls>
+			<JetpackFieldDimensionControls
+				clientId={ clientId }
+				setAttributes={ setAttributes }
+				width={ width }
+			/>
+			<InspectorControls group="styles">
+				<ToolsPanel
+					label={ __( 'Color', 'jetpack-forms' ) }
+					panelId={ clientId }
+					resetAll={ () => {
+						setAttributes( {
+							labelColor: undefined,
+							inputColor: undefined,
+							buttonBackgroundColor: undefined,
+							fieldBackgroundColor: undefined,
+							borderColor: undefined,
+						} );
+					} }
+					toolsPanelDropdownMenuProps={ toolsPanelDropdownMenuProps }
+				>
+					<div className="jetpack-field-controls__color-settings">
+						<ColorGradientSettingsDropdown
+							__experimentalIsRenderedInSidebar
+							settings={ colorSettings }
+							panelId={ clientId }
+							gradients={ [] }
+							disableCustomGradients
+							{ ...useMultipleOriginColorsAndGradients() }
+						/>
+					</div>
+				</ToolsPanel>
+				<ToolsPanel
+					label={
+						isChoicesBlock
+							? __( 'Options Typography', 'jetpack-forms' )
+							: __( 'Input Typography', 'jetpack-forms', 0 )
+					}
+					resetAll={ () => {
+						setAttributes( {
+							fieldFontSize: undefined,
+							lineHeight: undefined,
+							buttonBorderWidth: undefined,
+							buttonBorderRadius: undefined,
+							borderWidth: undefined,
+							borderRadius: undefined,
+						} );
+					} }
+					toolsPanelDropdownMenuProps={ toolsPanelDropdownMenuProps }
```

Comment:
```suggestion
					dropdownMenuProps={ toolsPanelDropdownMenuProps }
```

Typo: Incorrect prop name.